### DISCUSSION TITLE
Password utente amministratore

### DISCUSSION THREAD

**Discussion by giannic81** [indomus/forum#2117] [HUB: Home Assistant]
Non riesco ad accedere più ad HA, mi restituisce credenziali errate, fino a che accedevo da rete interna ok, ma appena ho fatto l'accesso da esterno (dopo aver configurato duckDNS) vuole il login e mi dice errato.
C'è un modo per recuperarlo?

**Comment by snakuzzo** [comment]
Se le avevi salvate sul browser, recuperale da lì...altrimenti leggi [QUI](https://www.home-assistant.io/docs/locked_out/)

**Comment by giannic81** [comment]
Grazie

Il giorno lun 28 nov 2022 alle 14:12 Diego Ambrosanio <
***@***.***> ha scritto:

> Se le avevi salvate sul browser, recuperale da lì...altrimenti leggi QUI
> <https://www.home-assistant.io/docs/locked_out/>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/indomus/forum/discussions/2117#discussioncomment-4253298>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A4K3EWA6XO7E5JVBEIPADLDWKSVVLANCNFSM6AAAAAASNJGI4E>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
-- 
* Cordialmente*
